plugins/catalog-react/src/filters.ts

```diff
@@ -331,3 +332,19 @@ export class EntityErrorFilter implements EntityFilter {
     return error !== undefined && this.value === error;
   }
 }
+
+/**
+ * Sort entities by a given field/column.
+ * @public
+ */
+export class EntityOrderFilter implements EntityFilter {
+  constructor(readonly value: EntityOrderQuery) {}
```

Comment:
If possible, it might be useful to make the filter serializable by allowing the constructor to accept a `string`
 or `string[]` as a value. This way, we can serialize the filter in the query parameters and reconstruct it on page load.
 
```suggestion
  constructor(readonly value: [string, 'asc' | 'desc'][]) {}
```